NuKeeper.Tests/Engine/RepositoryFilterTests.cs

```diff
@@ -50,7 +50,7 @@ public async Task ShouldNotFilterWhenSearchFails()
 
             var result = await subject.ContainsDotNetProjects(MakeSampleRepository());
 
-            Assert.True(result);
+            Assert.False(result);
```

Comment:
This was actually a bug. If you look at the previous versions of the contains function you will see that it will return true even if throws an exception. This test was never correct I believe

Reply:
I wish :-) . The use case is: I've got about 110 pushable repos in my GitHub organisation, after checking/updating about 70 of them, GitHub API starts throttling the searches and OktoKit throws exceptions. So, NuKeeper falls back to doing a full clone and checking for updates locally. (It still makes sense to try filtering out remotely first, it's much faster).